build/bancodedados/init_db.sql

```diff
@@ -5,44 +5,45 @@ USE didynDB;
 
 CREATE TABLE IF NOT EXISTS escola(
   id_escola INT(11) UNIQUE AUTO_INCREMENT,
-  nome VARCHAR(255),
+  nome VARCHAR(255) NOT NULL,
   endereco VARCHAR(255),
   PRIMARY KEY (id_escola)
+  -- test
 );
 
 CREATE TABLE IF NOT EXISTS aluno(
   id_aluno INT(11) UNIQUE AUTO_INCREMENT,
-  id_escola INT(11),
-  nome VARCHAR(255),
-  gub VARCHAR(255),
+  id_escola INT(11) NOT NULL,
+  nome VARCHAR(255) NOT NULL,
+  gub INT(11) NOT NULL,
   professor VARCHAR(255),
   PRIMARY KEY (id_aluno),
-  FOREIGN KEY (id_escola) REFERENCES escola(id_escola)
+  FOREIGN KEY (id_escola) REFERENCES escola(id_escola) ON DELETE CASCADE
 );
 
 CREATE TABLE IF NOT EXISTS avaliacao(
-  id_avalicao INT(11) UNIQUE AUTO_INCREMENT,
-  id_aluno INT(11),
-  mestre VARCHAR(255),
-  gub VARCHAR(255),
-  sagui INT(11),
-  sonkisul INT(11),
-  balkisul INT(11),
-  sibon_donjak_1 INT(11),
-  sibon_donjak_2 INT(11),
-  comb_tec_1 INT(11),
-  comb_tec_2 INT(11),
-  comb_tec_3 INT(11),
-  taeguk_1 INT(11),
-  taeguk_2 INT(11),
-  taeguk_3 INT(11),
-  matchuho_kyorugui INT(11),
-  kyorugui INT(11),
-  kyopa INT(11),
-  avalicao_periodica INT(11),
-  avalicao INT(11),
+  id_avaliacao INT(11) UNIQUE AUTO_INCREMENT,
+  id_aluno INT(11) NOT NULL,
+  mestre VARCHAR(255) NOT NULL,
+  gub VARCHAR(255) NOT NULL,
+  sagui INT(11) NOT NULL,
```

Comment:
Não acho que seja interessante fazer das notas NOT NULL. Pois quero que seja possível criar uma avaliação sem nota e a medida que o evento ocorrer, atribuir uma nota de cada vez